macx/scripts/osxdist.py

```diff
@@ -312,6 +312,7 @@ def package_client():
 			'release/Mumble.app/Contents/Plugins/libmanual.dylib',
 			'release/Mumble.app/Contents/Codecs/libcelt0.0.7.0.dylib',
 			'release/Mumble.app/Contents/Codecs/libcelt0.0.11.0.dylib',
+			'release/Mumble.app/Contents/Codecs/libopus.dylib',
```

Comment:
This change is unrelated to this commit? Split it?

Reply:
Thank you very much for noticing, this change should've been in b34ab4a4d4684874e0844db42921c2d82d30d45b.